src/axom/sidre/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ set(sidre_depends
     conduit::conduit
     )
 
+blt_list_append(TO sidre_depends ELEMENTS blt::hip IF ENABLE_HIP)
```

Comment:
Does the new build system ensure that HIP will be pulled in from Axom Core?  I don't think Sidre has a direct dependency on HIP.

Reply:
ALE3D also uses an old version of Axom, possibly not including the recent build fixes.  I'm inclined to remove this line and the line from slam's CMakeLists.txt.

Reply:
Removed.